packages/driver/src/cypress.js

```diff
@@ -649,7 +649,7 @@ $Cypress.prototype.Blob = blobUtil
 $Cypress.prototype.Promise = Promise
 $Cypress.prototype.minimatch = minimatch
 $Cypress.prototype.sinon = sinon
-$Cypress.prototype.lolex = lolex
+$Cypress.prototype.fakeTimers = fakeTimers
```

Comment:
This is where I was a bit in doubt. Should I either keep this as lolex, or do we need to document this change somehow as the comment says its used for monkeypatching?

Reply:
Let's keep it as `lolex` for now, especially since its API is the same. I don't think that it's documented, only [a few of the prototype utilities are](https://docs.cypress.io/api/utilities/_) (see the sidebar under Utilities), so it might not technically be breaking to change it. But I'd rather err on the safe side and we'll consider changing it with the next major release.

Reply:
Okay, I'm good with that!